### ISSUE TITLE
Python: need full error message

### ISSUE THREAD

**Issue by gpotter2** [numworks/epsilon#237]
We definatly need full error messages in the python app.

![image](https://user-images.githubusercontent.com/10530980/32690684-d5e7e782-c6fa-11e7-8dde-209da64e3804.png)
How useful...

By having a quick look at MicroPython source, this seems to be possible.

We also need a saving&opening file support (you definatly need to have more than one program at the same time), but I guess this will come later.

**Comment by adriweb** [comment]
Yes, teachers have been reporting this several times already - they said they're working on Python-related improvements for a future update ; hopefully soon :)

**Comment by gpotter2** [comment]
@adriweb Alright ! Should I close this then ?

**Comment by adriweb** [comment]
I guess it can be closed when the code/commits to fix that lands on GitHub :P

**Comment by artaxxx** [comment]
Thanks! Let's keep it open until it's fixed :)
